mlflow/deployments/databricks/__init__.py

```diff
@@ -70,7 +70,7 @@ def _call_endpoint(
         prefix: str = "/api/2.0",
         route: Optional[str] = None,
         json_body: Optional[Dict[str, Any]] = None,
-        timeout: int = MLFLOW_HTTP_REQUEST_TIMEOUT.get(),
+        timeout: Optional[int] = None,
```

Comment:
timeout value should be determined when this function is called, not when this function is defined.